command/commands/token.py

```diff
@@ -30,11 +35,13 @@ def __init__(self, db_facade, config):
         self.expiry = config.expiry
         self.signing_key = config.signing_key
 
-    def handle(self, _command, user_id):
+    def handle(self,
+               _command: str,
+               user_id: str) -> ResponseTuple:
         """Handle request for token."""
         logging.debug("Handling token command")
         try:
-            user = self.facade.retrieve(User, user_id)
+            user = cast(User, self.facade.retrieve(User, user_id))
```

Comment:
Mentioned this in the last pr, but could the casting be done at the `dynamodb` or `facade` level of `retrieve()`?

Reply:
I was wondering how the generics would work but then i figured it out and now you no longer have to explicitly cast it to something.

QED
YAY